### ISSUE TITLE
pytorch version and results reproduction

### ISSUE THREAD

**Issue by drazellan** [a514514772/DISE-Domain-Invariant-Structure-Extraction#9]
Hi
Thanks for your code!
I trying to reproduce the result  in the paper but i get at best 42.4 miou with a batch size of 2 on a P6000 nvidia card with your default setting.

By the way one's needs a card with at least 20Gb of memory to run your code, i try on a card with 16 Gb and i got a memory issue...

I use pytorch 1.2 and i see that you used pytorch 0.3.1 so maybe it's the reason for the performances degradation as i know there is a difference in the default behaviour in upsample between those 2 versions.
Have you try to re run the code with a newer version of pytorch than the 0.3.1?
Thanks

**Comment by hui-po-wang** [comment]
No, we only try our code on 0.3.1. If my memory serves me well, there is a difference between 0.3.1 and 0.4.1 such that the code behaves differently in terms of accuracy and memory usage. 

Unfortunately, I do not have available machines to test it for now. But, please feel free to point out problems or consider a PR.

**Comment by drazellan** [comment]
Thanks for your reply
I just lauch a learning with modification in upsample to use align_corners=True like in the 0.3.1 version


________________________________
De : Hui-Po Wang <notifications@github.com>
Envoyé : mercredi 11 décembre 2019 09:58
À : a514514772/DISE-Domain-Invariant-Structure-Extraction <DISE-Domain-Invariant-Structure-Extraction@noreply.github.com>
Cc : drazellan <drazellan@hotmail.com>; Author <author@noreply.github.com>
Objet : Re: [a514514772/DISE-Domain-Invariant-Structure-Extraction] pytorch version and results reproduction (#9)


No, we only try our code on 0.3.1. If my memory serves me well, there is a difference between 0.3.1 and 0.4.1 such that the code behaves differently in terms of accuracy and memory usage.

Unfortunately, I do not have available machines to test it for now. But, please feel free to point out problems or consider a PR.

**Comment by huitangtang** [comment]
@drazellan Hi, what's your result? I found that using align_corners=True produced a similar result. And when I run the code on two 2080Ti cards with 11 Gb, the result of miou degrades to <40%. Is it normal? Thank you for your response!
